eva-domain/src/main/kotlin/com/razz/eva/domain/Model.kt

```diff
@@ -18,4 +18,9 @@ abstract class Model<ID : ModelId<out Comparable<*>>, E : ModelEvent<ID>>(
 
     protected fun raiseEvent(newEvent: E): EntityState<ID, E> =
         entityState.raiseEvent(newEvent)
+
+    fun <T> proto(): T? {
```

Comment:
I really hate exposing this as a public fun. We could do some gradle magic to pass repo and model modules as a single compilation unit and making this fun `internal`.